### PR TITLE
fix!: RUN -> IST in bootstrap (vats, run-protocol)

### LINKED ISSUE #5328: Rename RUN to Inter (IST) in Bootstrap

### PR DESCRIPTION
**Pull request by dckc**
goal:

closes: #5328

## Description

Rename RUN -> IST in bootstrap

TODO:
 - [x] I got a little carried away with search-and-replace and did a bunch of stuff with contract keywords; I suppose I should back that out for now.
 - [ ] ensure loadgen works (with @mhofman )
 - [ ] ensure dApps with work (with @samsiegart )

### Security Considerations

_should_ have no security impact, but destabilizes the code somewhat.

### Documentation Considerations

probably a lot

### Testing Considerations

I wonder how this interacts with dApps, loadgen

### CODE REVIEW

**Comment by dckc** [comment]
@mhofman help me address the risk of breaking loadgen here, please?

File: packages/run-protocol/src/core-proposal.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { Stable } from '@agoric/vats/src/tokens.js';
```
Comment (turadg) [inline packages/run-protocol/src/core-proposal.js]: 👍

File: packages/run-protocol/src/demoIssuers.js [hunk 2]
```diff
@@ -72,6 +78,7 @@ export const showBrand = b =>
   `${b}`.replace(/.object Alleged: (.*) brand./, '$1');
 export const showAmount = ({ brand, value }) => {
   const b = `${showBrand(brand)}`;
+  assert(b in DecimalPlaces, b);
```
Comment (turadg) [inline packages/run-protocol/src/demoIssuers.js]: the message has little information. consider expanding or removing.

File: packages/run-protocol/test/test-gov-collateral.js [hunk 3]
```diff
@@ -19,13 +19,16 @@ import { extractCoreProposalBundles } from '@agoric/deploy-script-support/src/ex
...
+import { Stable } from '@agoric/vats/src/tokens.js';
 import { makeNodeBundleCache } from './bundleTool.js';
 import { setupBootstrap, setUpZoeForTest } from './supports.js';
 
 const { details: X } = assert;
 const dirname = url.fileURLToPath(new URL('.', import.meta.url));
 
-/** @type {import('ava').TestInterface<Awaited<ReturnType<makeTestContext>>>} */
+/** @typedef {Awaited<ReturnType<makeTestContext>>} GovTestContext */
+/** @typedef {import('ava').ExecutionContext<GovTestContext>} GovTestCtx */
```
Comment (turadg) [inline packages/run-protocol/test/test-gov-collateral.js]: Reads like it has to do with `TestInferface`. Consider spacing it out.

File: packages/vats/src/distributeFees.js [hunk 4]
```diff
@@ -40,10 +40,12 @@ export async function buildDistributor(
   } = params;
 
   async function schedulePayments() {
-    contracts.map(contract =>
-      E(contract)
-        .collectFees()
-        .then(payment => E(feeDepositFacet).receive(payment)),
+    await Promise.all(
```
Comment (turadg) [inline packages/vats/src/distributeFees.js]: good it's returning a promise now. could be simplified:
```
schedulePayments = () => Promise.all(
```

File: packages/zoe/src/zoeService/feeMint.js [hunk 5]
```diff
@@ -1,10 +1,19 @@
...
-import { makeIssuerKit } from '@agoric/ertp';
+import { makeIssuerKit, AssetKind } from '@agoric/ertp';
 
 import { makeHandle } from '../makeHandle.js';
 
 const { details: X } = assert;
+
+export const defaultFeeIssuerConfig = harden(
+  /** @type {const} */ ({
+    name: 'ZOE',
```
Comment (turadg) [inline packages/zoe/src/zoeService/feeMint.js]: consider something more conspicuous when debugging. perhaps `ZOED` or `ZDEFAULT`.

**Review by turadg — APPROVED** [review]
Thanks for DRYIng it out along the way. Will make it much easier the next time the protocol name changes ;-)

**Review by turadg — APPROVED** [review]

**Comment by dckc** [comment]
Based on discussion with @mhofman , it looks like updating loadgen for just this part of the renaming, without changing the contract keywords as well (#4800 ) is not cost-effective. So I'll let this wait until that part is ready too.

**Comment by mhofman** [comment]
Closing since this was rolled into #5585

**Pull request closed without merging**